server/base-routes.js

```diff
@@ -184,7 +187,7 @@ var baseRoutes = [
   }, {
     method: 'GET',
     path: '/api/client-env.js',
-    handler: function(request, reply) {
+    handler: function(request, h) {
```

Comment:
I see h is being used differently in this handler compared to the last.

Reply:
I could "promisify" that pollyfillio function above, but I chose instead to wrap it in a promise, since I don't really trust `util.promisify` that much. In this case, the request response is available to return without needing to await an async result, so we can just return it.